test/session.js

```diff
@@ -1,51 +1,53 @@
+/* global describe, it, before */
```

Comment:
oh that's fine I just was trying to figure out how to get rid of the standard error.